doc/whats_new.rst

```diff
@@ -6,6 +6,34 @@
 Release history
 ===============
 
+Version 0.18.2
+==============
+
+Changelog
+---------
+
+New features
+............
+
+   - Added a `'eleven-point'` interpolated average precision option to
+     :func:`metrics.ranking.average_precision_score` as used in the `PASCAL
+     Visual Object Classes (VOC) Challenge <http://citeseerx.ist.psu.edu/viewdoc/
+     download?doi=10.1.1.157.5766&rep=rep1&type=pdf>`_.
+     (`#7356 <https://github.com/scikit-learn/scikit-learn/pull/7356>`_). By
+     `Nick Dingwall`_
+
+
+Bug fixes
+.........
+
+   - :func:`metrics.ranking.average_precision_score` no longer linearly
+     interpolates between operating points, and instead weights precisions
+     by the change in recall since the last operating point, as per the
+     `Wikipedia entry <http://en.wikipedia.org/wiki/Average_precision>`_.
```

Comment:
I think you need to cite a more stable reference; at a minimum, a versioned Wikipedia page.